visidata/_open.py

```diff
@@ -108,6 +108,9 @@ def openPath(vd, p, filetype=None, create=False):
         vd.status('creating blank %s' % (p.given))
         return newfunc(p)
 
+    if p.is_dir():  # also matches symlinks to dirs
+        return vd.open_dir(p)
```

Comment:
This is breaking the vgit tests! They operate on a dir but load them with `--filetype` git. Is there any way to accomodate that usecase?

Reply:
Oops! Okay, I've modified it to ignore the extension only for directories, and only if no other filetype has been specified.